cogs/modmail.py

```diff
@@ -706,7 +740,12 @@ def format_log_embeds(self, logs, avatar_url):
     @trigger_typing
     @checks.has_permissions(manage_channels=True)
     async def unblock(self, ctx, *, user: User = None):
-        """Unblocks a user from using Modmail."""
+        """
+        Unblocks a user from using Modmail.
+
+        Note: reasons start with "System Message: " are reserved for internal
```

Comment:
`reasons that start with`

Reply:
The sentence in its current state is grammatically correct. The "that" is just personal preference. I personally like to exclude "that" when possible, but I guess I'll add it anyways.